assets/src/hooks/useDeviceSupportsHover.ts

```diff
@@ -0,0 +1,7 @@
+import { useMediaQuery } from "@react-hook/media-query"
+
+const useDeviceSupportsHover = (): boolean => {
+  return !useMediaQuery("(hover: none)")
```

Comment:
**question(non-blocking):** is there a particular reason to do this instead of `return useMediaQuery("(hover: hover)")`?